docs/OperationsAPI.yaml

```diff
@@ -0,0 +1,313 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Mobility Database Catalog Operations
+  description: |
+    API for the Mobility Database Catalog Operations. See [https://mobilitydatabase.org/](https://mobilitydatabase.org/). 
+    This API was designed for internal use and is not intended to be used by the general public.
+    The Mobility Database Operation API uses Auth2.0 authentication.
+  termsOfService: https://mobilitydatabase.org/terms-and-conditions
+  contact:
+    name: MobilityData
+    url: https://mobilitydata.org/
+    email: api@mobilitydata.org
+  license:
+    name: MobilityData License
+    url: https://www.apache.org/licenses/LICENSE-2.0
+
+tags:
+  - name: "operations"
+    description: "Mobility Database Operations"
+
+paths:
+  /v1/operations/feeds/gtfs:
+    put:
+      description: Update the specified GTFS feed in the Mobility Database.
+      tags:
+        - "operations"
+      operationId: updateGtfsFeed
+      security:
+        - ApiKeyAuth: []
+      requestBody:
+        description: Payload to update the specified GTFS feed.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateRequestGtfsFeed"
+      responses:
+        200:
+          description: >
+            The feed was successfully updated. No content is returned.
+        204:
+          description: >
+            The feed update request was successfully received, but the update process was skipped as the request matches with the source feed.
+        400:
+          description: >
+            The request was invalid.
+        401:
+          description: >
+            The request was not authenticated or has invalid authentication credentials.
+        500:
+          description: >
+            An internal server error occurred.
+
+components:
+  schemas:
+    Redirect:
+      type: object
+      properties:
+        target_id:
+          description: The feed ID that should be used in replacement of the current one.
+          type: string
+          example: mdb-10
+        comment:
+          description: A comment explaining the redirect.
+          type: string
+          example: Redirected because of a change of URL.
+    BasicFeed:
```

Comment:
Could it refer to the BasicFeed in DatabaseCatalogAPIyml?

Reply:
It will be tricky to do an import from `DatabaseCatalogAPI.ym` as it will impact the development process and the generator. However, a good improvement when we implement the getFeeds endpoint, add a BasicFeed entity to this schema.